### DISCUSSION TITLE
Psikyo Shooters -- Steam or Switch (Compilation) - Roms or Ports?

### DISCUSSION THREAD

**Discussion by CartCollector** [farmerbb/RED-Project#165] [Q&A]
Do we know if there is a chance the Psikyo Shooter collections on Switch, or the individual releases on Steam, are emulated with roms?  They look like they might be ported, using arcade assets, but I wasn't sure if anyone had looked into this.

**Comment by farmerbb** [comment]
Samurai Aces and Tengai contain loose ROMs, and can be zipped and run in emulators: https://github.com/farmerbb/RED-Project/discussions/93

I looked at the directory structure of the others on SteamDB, and it doesn't look like any ROMs are readily accessible.
